ES/ESum042.xml

```diff
@@ -30,608 +28,321 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-                                 <dim unit="leaf">8</dim>
-                                 <locus from="23r" to="30v"/> ፬IV(8/fols. 23r-30v) </item>
-                              <item xml:id="q5">
-                                 <dim unit="leaf">8</dim>
-                                 <locus from="31r" to="38v"/> ፭V(8/fols. 31r-38v) </item>
-                              <item xml:id="q6">
-                                 <dim unit="leaf">7</dim>
-                                 <locus from="39r" to="45v"/> ፮VI(7/fols. 39r-45v) </item>
-                              <item xml:id="q7">
-                                 <dim unit="leaf">11</dim>
-                                 <locus target="#11"/>
-                                 <locus from="46r" to="56v"/> VII(11/fols. 46r-56v) </item>
-                              <item xml:id="q8">
-                                 <dim unit="leaf">8</dim>
-                                 <locus from="57r" to="64v"/> ፰VIII(8/fols. 57r-64v) </item>
-                              <item xml:id="q9">
-                                 <dim unit="leaf">11</dim>
-                                 <locus target="#11"/>
-                                 <locus from="65r" to="75v"/> ፱IX(11/fols. 65r-75v) </item>
-                              <item xml:id="q10">
-                                 <dim unit="leaf">12</dim>
-                                 <locus target="#12"/>
-                                 <locus from="76r" to="87v"/> ፲X(12/fols. 76r-87v) </item>
-                              <item xml:id="q11">
-                                 <dim unit="leaf">16</dim>
-                                 <locus target="#16"/>
-                                 <locus from="88r" to="104v"/> ˂፩/፪/፫XI(16/fols. 88r-104v) </item>
-                              <item xml:id="q12">
-                                 <dim unit="leaf">13</dim>
-                                 <locus target="#13"/>
-                                 <locus from="105r" to="117v"/> ፪/፬/፭XII(13/fols. 105r-117v) </item>
-                              <item xml:id="q13">
-                                 <dim unit="leaf">8</dim>
-                                 <locus from="118r" to="125v"/> ፫/፮/፬XIII(8/fols. 118r-125v) ˃ </item>
-                              <item xml:id="q14">
-                                 <dim unit="leaf">6</dim>
-                                 <locus from="126r" to="132v"/> ˂XIV(6/fols. 126r-132v) </item>
-                              <item xml:id="q15">
-                                 <dim unit="leaf">6</dim>
-                                 <locus from="133r" to="138v"/> XV(6/fols. 133r-138v)˃ </item>
-                              <item xml:id="q16">
-                                 <dim unit="leaf">6</dim>
-                                 <locus from="139r" to="144v"/> ˂XVI(6/fols. 139r-144v)˃</item>
-                           </list>
+                        <list>
```

Comment:
test transformation report, FYI
Quire with id:q1 and n 1 is made of 7 leaves, but there is no futher information about the reason this number is odd.
Quire with id:q2 and n 2 is made of 7 leaves, but there is no futher information about the reason this number is odd.
Quire with id:q6 and n 6 is made of 7 leaves, but there is no futher information about the reason this number is odd.
Quire with id:q7 and n 7 is made of 11 leaves, but there is no futher information about the reason this number is odd.
Quire with id:q12 and n 12 is made of 13 leaves, but there is no futher information about the reason this number is odd.

Reply:
I think for q12 everything should be correct; for other quires the structure cannot be see exactly because they are hold by parchment guards.